server/src/test/scala/org/http4s/server/middleware/DateSpec.scala

```diff
@@ -9,7 +9,7 @@ import cats.effect.testing.specs2.CatsIO
 
 class DateSpec extends Http4sSpec with CatsIO {
 
-  implicit val T = IO.timer(scala.concurrent.ExecutionContext.global)
+  override implicit val timer: Timer[IO] = Http4sSpec.TestTimer
```

Comment:
I'm surprised this is necessary at all, but I'm also pretty sure the compiler said it was and it has a better memory than me.

Reply:
I think its because both CatsIO and Http4sSpec define implicit timers, so as a result we have 2 instances with the same priority so it fails to find either implicitly, by overriding in the trait we create a more specific instance of the implicit which takes priority over the inherited traits.

Reply:
`Http4sSpec` should probably be reworked to be built on top of `CatsIO`, but that's a whole other chore.